andela/balrog
# 1365658 fix priority background rate validators

#### What does this PR do?
This PR is to resolve [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1365658)
To supplement the backend validation hack that had been implemented earlier with front-end validation for Priority and Background Rate input fields with specific validation errors.
#### Description of Task to be completed?
Implement integer and range validation for Priority and Background rate inputs on the UI.
#### How should this be manually tested?
1. Once balrog is running on http://127.0.0.1:8080 navigate to `Rules` and click on `Add a New Rule` or `Update` on an existing rule. 
2. Test by inputting invalid data for `Rate`- either negative numbers(eg `-40`), over 100(eg 144) or non-digit characters(eg 'x', or '&').
3. Also, input invalid data for `Priority` - either negative numbers or non-digit characters. 
4. An error label will appear below the respective input with a relevant error message when the user clicks on the save changes button.
5. Repeat 2 - 3 after navigating to 'Rules/Scheduled Changes' and 'Rules/Scheduled Changes/Update'. 
 
#### What are the relevant bugzilla stories?
https://bugzilla.mozilla.org/show_bug.cgi?id=1365658
#### Screenshots (if appropriate)
<img width="1064" alt="screen shot 2017-10-03 at 14 35 00" src="https://user-images.githubusercontent.com/28535137/31123318-18e53bec-a848-11e7-8668-c14fcce2caf4.png">
